packages/amplify-category-storage/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@aws-amplify/amplify-category-storage",
-  "version": "3.7.4",
+  "version": "5.0.0-beta.6",
```

Comment:
why did we jump from 3 to 5?

Reply:
Lerna did this and we didn't catch it. I.e. it saw `feat!` PR two times at some point of releasing betas.
However, since we burned couple of 4.x.x-beta tags. It's not worth going back - i.e. we'd clash with 5.x betas if we tried to beta vNext. 6.x is then open for grab.